pyhap/accessory_driver.py

```diff
@@ -642,16 +642,18 @@ def persist(self):
         tmp_filename = None
         try:
             temp_dir = os.path.dirname(self.persist_file)
+            logger.debug(f"Creating temp persist file in {temp_dir!r}")
             with tempfile.NamedTemporaryFile(
                 mode="w", dir=temp_dir, delete=False
             ) as file_handle:
                 tmp_filename = file_handle.name
+                logger.debug(f"Created temp persist file {file_handle!r} named {tmp_filename!r}")
```

Comment:
Please use lazy formatting for loggers
https://pylint.readthedocs.io/en/stable/user_guide/messages/warning/logging-fstring-interpolation.html

Reply:
fixed it